starter/js/atm.js

```diff
@@ -1,12 +1,44 @@
-//Begin with the document ready function
-
-    //Checking account deposit function
+$( document ).ready(function() {
+    var availBalChecking = 0;
+    var availBalSavings = 0;
+    var transactionAmount = 0;
+    $('#depositChecking').click(function() {
+      transactionAmount = parseFloat($('#amountChecking').val());
+      availBalChecking += transactionAmount;
+      $('#checkingBalance').html("$" + availBalChecking);
+      });
```

Comment:
Indentation! Closing brackets should be lined up with the start of the line on which they're opened. Also, these listeners and functions should all line up with each other. #1 way to make code more readable!